### PR TITLE
Shader Dev Window & Other Stuff

### PR DESCRIPTION
**Pull request by ghost**
Continues https://github.com/space-wizards/RobustToolbox/pull/1145

### CODE REVIEW

**Comment by ghost** [comment]
https://github.com/space-wizards/RobustToolbox/pull/1145#discussion_r446159811
I'm not sure what you mean by this (PJB) (I'm not really familiar with css/webdev), but I fixed all the other issues that needed to be fixed.

**Comment by SweptWasTaken** [comment]
@PJB3005

File: Robust.Client/UserInterface/Controls/ColorSlider.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using Robust.Shared.Maths;
+
+namespace Robust.Client.UserInterface.Controls
+{
+
+    public sealed class ColorSlider : Control
```
Comment (PJB3005) [inline Robust.Client/UserInterface/Controls/ColorSlider.cs:7]: This feels way too specific to be part of the engine.

File: Robust.Client/UserInterface/Controls/Slider.cs [hunk 2]
```diff
@@ -64,8 +75,10 @@ public class Slider : Range
             }
         }
 
-        public Slider()
+        public Slider(string? styleClass = null)
         {
+            if(styleClass != null)
+                StyleClasses = new[] { styleClass };
```
Comment (PJB3005) [inline Robust.Client/UserInterface/Controls/Slider.cs:81]: This really should not be a thing.

File: Robust.Client/UserInterface/Controls/ColorSlider.cs (hunk 1 above)
Comment (Acruid) [reply Robust.Client/UserInterface/Controls/ColorSlider.cs:7]: Is this the beginning of https://github.com/space-wizards/RobustToolbox/issues/901?

**Comment by ghost** [comment]
Sorry, probably not going to ever finish this so gonna officially abandon it.

**Pull request closed without merging**